packer/github-actions-runner/sources.pkr.hcl

```
@@ -9,7 +9,9 @@ source "amazon-ebs" "github-actions-runner" {
   iam_instance_profile                      = "bcda-mgmt-github-actions"
 
   source_ami_filter {
-    filters = { name = "${var.ami_filter}" }
+     filters = {
+      name = "al2023-legacy-gi-2024-07-10T10-40-52Z" 
```

The filter should allow for new images to be selected each month:

```suggestion
      name = "al2023-legacy-gi-*" 
```

Also, since we're hardcoding the filter here instead of referencing `var.ami_filter` (which I'm in favor of doing) we can drop the `PKR_VAR_ami_filter` param in the workflow and remove the variable from packer/github-actions-runner/variables.pkr.hcl.